packages/flutter/test/widgets/custom_multi_child_layout_test.dart

```diff
@@ -90,6 +91,87 @@ class NotifierLayoutDelegate extends MultiChildLayoutDelegate {
   }
 }
 
+// LayoutDelegate that layout child with id 0 and 1
+// Used in the 'performLayout error control test' test case for triggering
+// For triggering an error when laying out non existent child
```

Comment:
"For triggering an error when laying out non existent child" ==> "an error when laying out a non existent child"